tests/files/packet_fedora39-calico-selinux.yml

```diff
@@ -0,0 +1,14 @@
+---
+# Instance settings
+cloud_image: fedora-39
+mode: default
+
+# Kubespray settings
+auto_renew_certificates: true
+# Switching to iptable due to https://github.com/projectcalico/calico/issues/5011
+# Kubernetes v1.23.0 kube-proxy does use v.7.x now. Calico v3.20.x/v3.21.x Pods show the following error
+# Bad return code from 'ipset list'. error=exit status 1 family="inet" stderr="ipset v7.1: Kernel and userspace incompatible: settype hash:ip,port with revision 6 not supported by userspace.
+kube_proxy_mode: iptables
```

Comment:
The linked issue is closed, do we still need to keep this ?
If not, we should drop it now.

Reply:
I pushed new commits. We can drop it if the CI test passes.